pupil_src/shared_modules/pupil_detector_plugins/manager.py

```diff
@@ -0,0 +1,75 @@
+import typing as T
+
+from pyglui import ui
+
+from plugin import Plugin
+
+from . import available_detector_plugins
+
+
+class PupilDetectorManager(Plugin):
+    order = 0.2
+
+    def __init__(self, g_pool):
+        super().__init__(g_pool)
+
+        self._available_pupil_detector_classes = available_detector_plugins()[1]
```

Comment:
I think this improves readability. The previous version made me think that the property was set to the plugin at index `1`, which seemed arbitrary. This version makes it clear that the property is set to the second value returned by the function.

```suggestion
        _, self._available_pupil_detector_classes = available_detector_plugins()
```

Reply:
Done!